packages/react-select/src/components/Menu.tsx

```diff
@@ -690,7 +684,7 @@ export const MenuPortal = <
   );
 
   return (
-    <PortalPlacementContext.Provider value={{ getPortalPlacement }}>
+    <PortalPlacementContext.Provider value={{ setPortalPlacement }}>
```

Comment:
State setters are equal between renders, but the object will be created each time. Might be worth memoizing to reduce consumer re-renders?

```tsx
const context = useMemo(() => ({ setPortalPlacement }), [setPortalPlacement])

return <PortalPlacementContext.Provider value={context} />
```

Reply:
nice catch! 🎉